scripts/lint/spec_lint.py

```
@@ -0,0 +1,74 @@
+        return json.dumps(v, ensure_ascii=False)
+    raise SystemExit(f"ERROR: unsupported type {type(v)}")
+
+def preimage(entry):
+    e = dict(entry)
+    e.pop('attestations', None)
+    return canonical(e).encode('utf-8')
+
+def b3_hex(data: bytes) -> str:
+    try:
+        import blake3
+    except Exception:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)